automatic/bluegriffon/tools/chocolateyInstall.ps1

```diff
@@ -1,16 +1,16 @@
 ﻿$ErrorActionPreference = 'Stop'
 
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
 $packageArgs = @{
   packageName    = 'bluegriffon'
   fileType       = 'exe'
-  softwareName   = 'BlueGriffon*'
-
-  checksum       = 'af87a13bfa255f996a857f1936713279d27345b0ac2f8503ea312a82d39c10b7'
-  checksumType   = 'sha256'
-  url            = 'http://bluegriffon.org/freshmeat/2.3.1/bluegriffon-2.3.1.win32-installer.exe'
-
-  silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
+  file           = "$toolsPath\"
+  softwareName   = 'bluegriffon*'
+  silentArgs     = "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /LOG=`"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).InnoSetup.log`""
   validExitCodes = @(0)
 }
 
-Install-ChocolateyPackage @packageArgs
+Install-ChocolateyInstallPackage @packageArgs
+
+Remove-Item -Force -ea 0 "$toolsPath\*.exe","$toolsPath\*.ignore"
```

Comment:
```
ls $toolsPath\*.exe | % { rm $_ -ea 0; if (Test-Path $_) { sc "$_.ignore" } }
```

Reply:
I believe that the script doesn't continue after uninstall until after the installer have completely finished, which would make that redundant.

But I could change it anyhow.